packages/graphql-api/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could the base tsconfig be moved to a package as well similar to the tslint config?